### DISCUSSION TITLE
Why is the OneKey Application not available on my phone's store?

### DISCUSSION THREAD

**Discussion by MarekPasnikowski** [OneKeyHQ/app-monorepo#3779] [Q&A]
Hello.

I am writing here as I do not believe my issue is not actually a bug.

I use a phone with Sailfish Operating System installed on it, and thus changing a phone is not really an option for me.

What is the technical reason for lack of the application in my store?  Is it 64-bits or API version?

**Comment by revan-zhang** [comment]
HI @MarekPasnikowski , for mobile version, currently onekey app only support iOS & Android.We will update other mobile systems and stores based on user usage.Thanks.
